integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/context/ChoreoConnectImpl.java

```diff
@@ -129,26 +134,58 @@ void createTmpMgwSetup() throws CCTestException {
     }
 
     public void addCcLoggersToEnv() {
-        Logger enforcerLogger = LoggerFactory.getLogger("Enforcer");
-        Logger adapterLogger = LoggerFactory.getLogger("Adapter");
-        Logger routerLogger = LoggerFactory.getLogger("Router");
-        Logger mockBackendLogger = LoggerFactory.getLogger("MockBackend");
-        Slf4jLogConsumer enforcerLogConsumer = new Slf4jLogConsumer(enforcerLogger);
-        Slf4jLogConsumer adapterLogConsumer = new Slf4jLogConsumer(adapterLogger);
-        Slf4jLogConsumer routerLogConsumer = new Slf4jLogConsumer(routerLogger);
-        Slf4jLogConsumer mockBackendLogConsumer = new Slf4jLogConsumer(mockBackendLogger);
-        environment.withLogConsumer("enforcer", enforcerLogConsumer)
-                .withLogConsumer("adapter", adapterLogConsumer)
-                .withLogConsumer("router", routerLogConsumer)
-                .withLogConsumer("mockBackend", mockBackendLogConsumer);
+        environment.withLogConsumer("enforcer", getLogConsumer("Enforcer"))
+                .withLogConsumer("adapter", getLogConsumer("Adapter"))
+                .withLogConsumer("router", getLogConsumer("Router"))
+                .withLogConsumer("mockBackend", getLogConsumer("MockBackend"));
         if (Boolean.parseBoolean(System.getenv(ENFORCER_DEBUG_ENV))) {
             environment.withEnv("JAVA_OPTS", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5006");
         }
     }
 
+    private Slf4jLogConsumer getLogConsumer(String loggerName) {
+        Logger logger = LoggerFactory.getLogger(loggerName);
+        Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(logger);
+        return logConsumer;
+    }
+
     public static void addCustomJwtTransformer() throws CCTestException {
         Utils.copyFile(System.getProperty("jwt_transformer_jar"),
                 Utils.getTargetDirPath() + TestConstant.CC_TEMP_PATH + TestConstant.DROPINS_FOLDER_PATH
                         + File.separator + "jwt-transformer.jar");
     }
+
+    public static void addInterceptorCertToRouterTruststore() throws IOException {
+        String routerTruststore = System.getProperty("cc_root_dir") + TestConstant.RESOURCES_DIR
+                 + TestConstant.SECURITY_DIR + TestConstant.CA_CERTS_FILE;
```

Comment:
@praminda, I think we have some plans to remove this file in the future right? shall we copy this file to test resources, WDYT?
file: https://github.com/wso2/product-microgateway/blob/21db96ea9c6f3f4f0ee3ec482cc16b7363d7a03e/resources/security/ca-certificates.crt

Reply:
Thanks for noticing. Fixed with https://github.com/wso2/product-microgateway/pull/2611/commits/6ff8e7d55d85dc362095542cce2f14efbdcfd023